apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/class-starter-page-templates.php

```diff
@@ -104,6 +104,18 @@ public function register_meta_field() {
 			},
 		);
 		register_meta( 'post', '_starter_page_template', $args );
+
+		$args = array(
+			'type'           => 'string',
+			'description'    => 'Selected category',
+			'single'         => true,
+			'show_in_rest'   => true,
+			'object_subtype' => 'page',
+			'auth_callback'  => function () {
+				return current_user_can( 'edit_posts' );
```

Comment:
Shouldn't this be `edit_pages`, as we're adding `post_meta` to a page?
```suggestion
				return current_user_can( 'edit_pages' );
```